.husky/pre-commit

```diff
@@ -1,2 +1,3 @@
 pnpm format
 pnpm lint
+uvx ruff format --check templates/
```

Comment:
Check this document to install uv and uvx cli: https://docs.astral.sh/uv/getting-started/installation/